components/developer/clang-12/Makefile

```diff
@@ -0,0 +1,117 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+# Copyright 2019 Alexander Pyhalov
+# Copyright 2019 Michal Nowak
+# Copyright 2019 Till Wegmüller
+#
+
+BUILD_BITS=		64
+BUILD_STYLE=		cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		clang-12
+LLVM_NAME=		llvm
+CLANG_NAME=		clang
+
+COMPONENT_REVISION=	1
+COMPONENT_MAJOR_VERSION= \
+	$(shell echo $(COMPONENT_VERSION) | $(GSED) -e 's/\([0-9]\+\.[0-9]\+\).*/\1/')
```

Comment:
You seem to have misunderstood me: We need COMPONENT_VERSION but not necessarily the COMPONENT_REVISION!
COMPONENT_REVISION is the part of the full version number that is under our control and only needed to be set (or incremented) when we make local changes. Thus, when we introduce a new component or update to a newer version of an existing component, we don't need to set COMPONENT_REVISION.

Reply:
Oh... sorry, I just woke up...